commands/cmdutils/cmdutils.go

```diff
@@ -174,6 +174,43 @@ func LabelsPrompt(response *[]string, apiClient *gitlab.Client, repoRemote *glre
 	return nil
 }
 
+func MilestonesPrompt(response *int, apiClient *gitlab.Client, repoRemote *glrepo.Remote) (err error) {
+	var addMilestones bool
+	err = prompt.Confirm(&addMilestones, "Do you wish to attach a milestone?", true)
+	if err != nil {
+		return err
+	}
+	if addMilestones {
+		var milestoneOptions []string
+		milestoneMap := map[string]*gitlab.Milestone{}
+
+		lOpts := &gitlab.ListMilestonesOptions{}
+		lOpts.PerPage = 100
+		milestones, err := api.ListMilestones(apiClient, repoRemote.FullName(), lOpts)
```

Comment:
Shouldn't we consider the `state` of the milestones 🤔? This will return both `active` and `closed` milestones

https://docs.gitlab.com/ee/api/milestones.html#list-project-milestones

Reply:
I guess that makes sense, at first I didn't because -m also doesn't discriminate between `active` and `closed` ones, but in an interactive setting it makes sense to cut down on the amount of milestones and have just the ones that are `active`.

In the future if someone asks we can **maybe** add a flag that makes it consider `closed` one too.

Thanks for the insight I'll change it prompty

Reply:
Updated, please check the new code.

Reply:
For now, if a user wants to add a closed milestone, they can use `-m` but for the prompt, it's better we cut down the list as you rightly stated

Reply:
Yes I have modifier the opts to include `State: gitlab.String("active")` so the API will only return `active` milestones.